code/gbq/gbq_qr.py

```diff
@@ -27,6 +27,10 @@ def make_parser():
     parser.add_argument('--short_run',
                         help='Flag to do a short run; overrides model-default num_bags to 10 and uses 3 quantile levels',
                         action='store_true')
+    parser.add_argument('--output_root',
+                        help='Path to a directory in which model outputs are saved',
+                        type=lambda s: Path(s),
+                        default=Path('../../submissions-hub/model-output'))
```

Comment:
primary goal of this argument is to enable the integration test to run without overwriting existing files.